File: wcserver/c#/examples/BSMiniRPTC#.NET/frmSplash.cs
using System;
using System.Collections;
using System.Diagnostics;
using System.Drawing;
using System.Windows.Forms;

namespace BSMiniRPT
{
	internal partial class frmSplash : System.Windows.Forms.Form
	{


		private void frmSplash_Load(object eventSender, System.EventArgs eventArgs)
		{

			modBSMiniRPT.CenterForm(this);

			this.Refresh();

		}


    }
} //end of root namespace

File: wcserver/vb.net/examples/BSMiniRPT.NET/frmSplash.vb
Option Strict On
Option Explicit On

Friend Class frmSplash

    Inherits System.Windows.Forms.Form

    Private Sub frmSplash_Load(ByVal eventSender As System.Object, ByVal eventArgs As System.EventArgs) Handles MyBase.Load

        CenterForm(Me)

        Me.Show()
        Me.Refresh()

    End Sub

End Class